forem/forem
# Standalone "admin" page for each comment

## What type of PR is this? (check all applicable)


## Description

Currently we show a list of comments to admin on `admin/content_manager/comments`. 
There are mainly two changes in this PR:
1. Add `View Details` to each comment in the above page.
2. Click on `View Details` should open individual comment page, example: `admin/content_manager/comments/17`.

## Related Tickets & Documents

- Closes #19490 

## QA Instructions, Screenshots, Recordings

**How to test?**
1. Visit `admin/content_manager/comments` page as admin
2. Click on `View Details` and `View Comment` on any comment and see how the links work
3. On individual comment page, notice that `View Details` button is hidden.

<img width="1508" alt="Screenshot 2023-06-27 at 4 11 57 PM" src="https://github.com/forem/forem/assets/9396084/96afb508-6826-4f63-b9dc-d6e49734ef24">
<img width="1512" alt="Screenshot 2023-06-27 at 4 12 46 PM" src="https://github.com/forem/forem/assets/9396084/d5bb8d8b-7003-4235-829b-410365abcaed">


### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_

## Added/updated tests?
